libfreerdp/CMakeLists.txt

```diff
@@ -18,6 +18,17 @@
 set(MODULE_NAME "freerdp")
 set(MODULE_PREFIX "FREERDP")
 
+option(FREERDP_USE_VENDOR_PRODUCT_CONFIG_DIR "Use <vendor>/<product> path for resources" OFF)
+	set(FREERDP_RESOURCE_ROOT ${CMAKE_INSTALL_FULL_DATAROOTDIR})
```

Comment:
Hum strange indentation or is it the github view ?

Reply:
no, that is one tab too much :)